pkg/agent/prommetrics.go

```diff
@@ -76,6 +194,24 @@ func makePrometheusParts(globalstate *agentState) (PromMetrics, *prometheus.Regi
 		)),
 	}
 
+	// Some of of the metrics should have default keys set to zero. Otherwise, these won't be filled
+	// unil the value is non-zero (because something's happened), which makes it harder to
+	// distinguish between "valid signal of nothing" vs "no signal".
+	metricsWithDirection := []resourceChangePair{
+		// sched requested/approved
+		metrics.schedulerRequestedChange, metrics.schedulerApprovedChange,
+		// informant requested/approved
+		metrics.informantRequestedChange, metrics.informantApprovedChange,
+		// neonvm requested
+		metrics.neonvmRequestedChange,
+	}
```

Comment:
why not put hard returns after every comma? I think that would be clear too?

Reply:
Done